dotcom-rendering/src/components/FormatBoundary.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { paletteDeclarations } from '../palette';
+import { useConfig } from './ConfigContext';
+
+type Props = {
+	format: ArticleFormat;
+	children: JSX.Element;
+};
+
+/**
+ * Used to create a set of palette colours for a different format
+ * to that of the :root element
+ */
+export const FormatBoundary = ({ format, children }: Props) => {
+	const { darkModeAvailable } = useConfig();
+
+	return (
+		<div
+			data-format-theme={format.theme}
+			data-format-design={format.design}
+			data-format-display={format.display}
+			css={[
+				css`
+					display: contents;
```

Comment:
This ensures that the element is not rendered itself, [see `display-box` on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/display-box). Accessibility concerns are nonexistent here as we do not want this element in the accessibility tree.